deploy/regression-test-container/docker-compose.yml

```diff
@@ -0,0 +1,112 @@
+version: '3'
+services:
+  sqldb:
+    image: "mcr.microsoft.com/mssql/server:2019-CU6-ubuntu-16.04"
```

Comment:
This sql version is getting out of date.  The latest is CU14 : https://hub.docker.com/_/microsoft-mssql-server

Reply:
ive created ticket 50650 to cover the work